tests/telemetry/conftest.py

```diff
@@ -171,6 +171,10 @@ def test_eventd_healthy(duthosts, enum_rand_one_per_hwsku_hostname, ptfhost, set
 
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
 
+    status = duthost.get_feature_status()
```

Comment:
Have we verified the fix? Shall we check the return status first of get_feature_status? 
"return feature_status, True"
https://github.com/sonic-net/sonic-mgmt/blob/0c3191f63370a96cc5fdd16ba16fe6c13f5871a9/tests/common/devices/sonic.py#L1506

Reply:
Yes, this should be fixed. It looks like that `'eventd' not in status` is always True for my current change.

Reply:
I mean we need to verify on both device, one for eventd enabled and another for disabled.

Reply:
It has been fixed.
It has been tested on two platform, one for eventd enabled and another for disabled.